src/blog/how-to-setup-uli-locally.mdx

```diff
@@ -72,7 +72,7 @@ http://localhost:3000 ws://localhost
 Your `content_security_policy` in the `manifest.json` should now look like this. 
 ```json
     "content_security_policy": {
-        "extension_pages": "default-src 'none'; connect-src https://ogbv-plugin.tattle.co.in/ https://uli-media.tattle.co.in/; font-src https://fonts.gstatic.com; object-src 'none'; script-src 'self'; style-src https://fonts.googleapis.com 'self' 'unsafe-inline'; img-src https://uli-media.tattle.co.in/; base-uri 'none'; form-action 'none'; frame-ancestors 'none'; report-uri 'none';"
+        "extension_pages": "default-src 'none'; connect-src http://localhost:3000 ws://localhost https://ogbv-plugin.tattle.co.in/ https://uli-media.tattle.co.in/; font-src https://fonts.gstatic.com; object-src 'none'; script-src 'self'; style-src https://fonts.googleapis.com 'self' 'unsafe-inline'; img-src https://uli-media.tattle.co.in/; base-uri 'none'; form-action 'none'; frame-ancestors 'none'; report-uri 'none';"
```

Comment:
can you explain why you added this to the file? I think @duggalsu removed this on purpose since this is a security risk if released on the extension stores. I think not having this in the repo and adding it only for local dev (as mentioned in the documentation) is safer too.

Reply:
no no, I havent made any changes to the actual package.json, this is just the change in the blog, that when setting up Uli locally for development, a user has to add `http://localhost:3000 ws://localhost` and then how should your `content_security_policy` look like. This is the blog content. 

Should this be not there too?

Reply:
aah! my bad. I was reviewing and only saw the diffed portion. didn't realize this was in the mdx file and not manifest. merging.